### PR TITLE
setup claim sct dialog

### PR DESCRIPTION
**Pull request by leopoldjoy**

### CODE REVIEW

**Comment by leopoldjoy** [comment]
Not ready to merge yet.

**Comment by bneiluj** [comment]
We need to describe the steps here about how to test this PR, should we create a running cycle on https://rinkeby.aragon.org/#/0x5a34C8f600027EeB82D23d2532F751dcBBeD4aA6/0x0b305085d0a510ff8e304c99ea0a3e2f7d094acb/ ? 
Screenshot 2020-04-13 at 11 36 10

**Comment by leopoldjoy** [comment]
1. Ensure that you’ve staked some SCT via the StakeDAO dashboard UX (stake dialog).

2. Navigate to the Aragon dashboard: https://rinkeby.aragon.org/#/0xf08bB06A2BF24c4b8be710181434792E18355371/0x5cd82077871305361b91d0b2b94f02b3b7c30e75/

3. Navigate to Stablecoin Rewards app.

4. Enter an amount (significantly) less than the "App Stablecoin Balance" in the input next to the "Distribute Reward" button.

5. Press the "Distribute Reward" and confirm the MetaMask dialogs.

6. Once confirmed, you can now claim the cycle via the StakeDAO dashboard UX.

**Comment by bneiluj** [comment]
Screenshot 2020-04-14 at 18 13 33
Screenshot 2020-04-14 at 18 12 45

`ALERT: Gas limit must be at least 21000` issue

File: src/screens/MintrActions/Claim/Complete.js [hunk 1]
```diff
@@ -24,15 +33,18 @@ const Success = ({ t, onDestroy, feesAvailable, networkName, transactionHash })
 							{t('mintrActions.claim.confirmation.actionDescription')}
 						</DataHeaderLarge>
 						<Amount>
-							{feesAvailable && feesAvailable[0] ? formatCurrency(feesAvailable[0]) : 0} sUSD
+							{Math.round(parseFloat(earnedRewardsBalance) * 1000000000000000000) /
```
Comment (bneiluj) [inline src/screens/MintrActions/Claim/Complete.js:36]: why is it static value ?

File: src/screens/MintrActions/Claim/index.js [hunk 2]
```diff
@@ -161,9 +242,10 @@ const Claim = ({ onDestroy }) => {
 		onClaim,
 		onClaimHistory,
 		goBack: handlePrev,
-		feesByPeriod,
+		claimableRewardsByCycle,
 		feesAreClaimable,
-		feesAvailable,
+		// feesAvailable,
```
Comment (bneiluj) [inline src/screens/MintrActions/Claim/index.js:247]: is `feesAvailable` just releated to SNX contracts ?

Comment (leopoldjoy) [reply src/screens/MintrActions/Claim/index.js:247]: Right

File: src/screens/MintrActions/Claim/Complete.js (hunk 1 above)
Comment (leopoldjoy) [reply src/screens/MintrActions/Claim/Complete.js:36]: What do you mean? `earnedRewardsBalance`? It's the total earnings for the claimable periods

**Comment by bneiluj** [comment]
Screenshot 2020-04-15 at 11 28 36


The `gas limit` is not passed into the smartcontract call

**Comment by bneiluj** [comment]
Screenshot 2020-04-15 at 12 07 32

**Pull request merged**